src/main/java/org/cryptomator/cryptofs/CipherDir.java

```diff
@@ -0,0 +1,14 @@
+package org.cryptomator.cryptofs;
+
+import java.nio.file.Path;
+import java.util.Objects;
+
+//own file due to dagger
+public record CipherDir(String dirId, Path contentDirPath) {
```

Comment:
i'll revert it to `path` for a small diff and consistency.

`c9rDirPath` is, from my point of view, the most misleading, because the directory with the `c9r` suffix contains the dir.c9r file with the dir-id. For a cleartext directory `/foo/bar` there are always two directories on the cipher side:
* the "linking" directory with the encrypted dir name, the c9r suffix and the dir-id as content
* the "content" directory with all encrypted files and other "linking" directories

Reply:
Also, this class cannot be package private. It is used in different packages, e.g. `DirectoryStreamFactory` or `MissingContentDir`.